core/src/main/java/tachyon/Format.java

```diff
@@ -26,18 +29,19 @@
  * Format Tachyon File System.
  */
 public class Format {
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
   private static final String USAGE = "java -cp target/tachyon-" + Version.VERSION
       + "-jar-with-dependencies.jar tachyon.Format <MASTER/WORKER>";
 
   private static boolean formatFolder(String name, String folder) throws IOException {
     UnderFileSystem ufs = UnderFileSystem.get(folder);
-    System.out.println("Formatting " + name + ": " + folder);
```

Comment:
These changes are unrelated to this PR. Should be done by another PR, which can improve the review process efficiency.